CONTRIBUTING.md

```diff
@@ -55,9 +54,7 @@ PRs that fix issues should include a reference like `Closes #XXXX` in the
 commit message so that github will automatically close the referenced issue
 when the PR is merged.
 
-<!--
-All PRs require at least two LGTMs (Looks Good To Me) from maintainers.
--->
+Most PRs will be reviewied by at least one [code owner][code-owners] (listed [here](.github/CODEOWNERS)).
```

Comment:
I think we should retain the 2 LGTM rule and be mindful while merging. I think we are following that almost all of the time.

Reply:
> I think we should retain the 2 LGTM rule...

The old LGTM line was commented out.  I can uncomment and/or add similar wording.  Is it two LGTMs from `OWNERS`?  From the relevant `CODEOWNERS`?  Reset when the PR's tip hash changes or not?  Is there anything like `Rejected` or `NACK` for votes against?  Are authors recused (would be difficult to get two LGTMs from `CODEOWNERS` with this restriction).

There's also a "reviewied" typo in my addition; I'll push a fix soon.

Reply:
More on GitHub's built in [review-requirement model][1], in case that affects your decisions.

[1]: https://help.github.com/articles/about-required-reviews-for-pull-requests/

Reply:
> I think we are following that almost all of the time.

I don't have numbers, but the last three PRs to land in master are:

* #1416, no explicit LGTMs or GitHub approvals (presumably @rhatdan merging is an implicit LGTM).
* #835, @mrunalp (codeowner) LGTM on the landed commit and an @rhatdan LGTM on an earlier version.
* #1412, @rhatdan LGTM.

That feels closer to “we'll probably get a codeowner LGTM/approval” or “we'll have at least one `OWNERS` LGTM/approval” than a two-LGTM requirement.

Reply:
We can formalize on TWO OWNERS LGTM if that is what people wants.  My only problem with that is in some cases it is a waste of peoples time.  I usually wait for Two LGTM from people.  The larger the Risk situations, I rely on the Owners.

Reply:
> We can formalize on TWO OWNERS LGTM…

I'm fine leaving it wiggly or making it strict; I just want the docs somewhat close to reality ;).  For wiggly two-`OWNERS`-LGTM, how about:

```markdown
Most PRs will be reviewed by two [approvers][prow-approvers] (listed [here](OWNERS)).
…
[prow-approvers]: https://godoc.org/k8s.io/test-infra/prow/repoowners#RepoOwners.Approvers
```

If we go that route, I think it's even more important to [describe the relationship between `OWNERS` and `CODEOWNERS`][1], which I still don't understand.

[1]: https://github.com/kubernetes-incubator/cri-o/pull/1272#issuecomment-359226481

Reply:
Sounds good to me.  I also have no idea what the difference is.